projects/ad9361/src/parameters.h

```diff
@@ -152,16 +152,13 @@
 #define CF_AD9361_RX_DMA_BASEADDR	2
 #define CF_AD9361_TX_DMA_BASEADDR	3
 
-#define SPI_DEVICE_ID		0
+#define SPI_DEVICE_ID		1
```

Comment:
The SPI_DEVICE_ID change is unrelated to this commit.
By the way, was this change needed for your setup? What .dts did you use?

Reply:
Thank you for your comment. You are right, it's a mistake. I will change it.

Device tree was generated with Xilinx tools from fmcomms2 project in [analogdevicesinc/hdl](https://github.com/analogdevicesinc/hdl/tree/master/projects/fmcomms2)

I havent tried but I guess it works as expected if you use device tree from [analogdevicesinc/linux](https://github.com/analogdevicesinc/linux/) e.g. zynq-zed-adv7511-ad9361-4-fmcomms2-3-4-userspace.dts